src/java.base/share/classes/java/lang/String.java

```diff
@@ -1957,9 +1960,14 @@ public boolean contentEquals(CharSequence cs) {
      * @see  #codePoints()
      */
     public boolean equalsIgnoreCase(String anotherString) {
+        if (anotherString != null) {
+            return false;
```

Comment:
This one is definitely wrong.

Reply:
Thank you

Reply:
So that begs the question as to what testing you did for this change?